### ISSUE TITLE
Regarding the Initialization of Model Soups 70 Checkpoints

### ISSUE THREAD

**Issue by alirezaabdollahpour** [naver-ai/model-stock#3]
# Regarding the Initialization of Model Soups 70 Checkpoints

Thank you for publishing this interesting paper. I have a question about the initialization choice for the 70 checkpoints provided in the Model Soups paper.

When using these checkpoints and setting their initialization as model_0, I noticed a discrepancy in baseline accuracy. The ImageNet accuracy for model_0 as initialization when trained with Linear Probing (LP) is 75%, whereas the zero-shot CLIP (ViT B/32) achieves only 64% accuracy.

This raises two important questions:

1. How did you determine which checkpoints were fine-tuned with LP initialization versus those fine-tuned with zero-shot?

2. When you mention using 48 checkpoints to achieve Uniform Soups, could you specify the exact indices of these 48 checkpoints among the full set of 70?

This information would help clarify the methodology and enable more accurate reproduction of your results.

**Comment by hellbell** [comment]
Hi, @alirezaabdollahpour. Thanks for your interest in our work.

> How did you determine which checkpoints were fine-tuned with LP initialization versus those fine-tuned with zero-shot?

Our work (model stock) follows [model soups](https://github.com/mlfoundations/model-soups)'s setting, which considers two initialized CLIP checkpoints. 
- The first one is zero-shot CLIP (without any further training)
- The other is LP-initialized CLIP (training linear probing layer while freezing the CLIP encoder [[See ref]](https://arxiv.org/abs/2202.10054))

Then, we fine-tune those initialized checkpoints multiple times.
- From zero-shot CLIP checkpoint, we conduct 48 fine-tunings with different hyper-paramter (following the model soups paper, but they didn't provide these fine-tuned checkpoints. We will release them soon) 
- From LP-initialized CLIP checkpoint, we conduct 70 fine-tunings with different hyper-paramter (these are already provided in [model soups repo](https://github.com/mlfoundations/model-soups))

> When you mention using 48 checkpoints to achieve Uniform Soups, could you specify the exact indices of these 48 checkpoints among the full set of 70?

Sorry for the confusion. As mentioned above, zero-shot 48 ckpts and LP-inited 70 ckpts are different. 
We will release the 48 fine-tuned checkpoints from zero-shot CLIP as soon as possible.

**Comment by alirezaabdollahpour** [comment]
Hello, I would greatly appreciate it if you could please upload the checkpoints to Google Drive or any other platform that's convenient for you, so they can be downloaded. It would be a great favor if you could do this as soon as possible.

**Comment by alirezaabdollahpour** [comment]
Hello again, I would greatly appreciate it if you could please upload the checkpoints to Google Drive or any other platform that's convenient for you, so they can be downloaded. It would be a great favor if you could do this as soon as possible.

**Comment by hellbell** [comment]
@alirezaabdollahpour 
I've uploaded the 48 models on Dropbox, you can access via [this link](https://www.dropbox.com/scl/fi/qd4xfvtfr2a6fbx88fjcn/zeroshot_ft_models.tar?rlkey=sc0zev6ydptl4pg7o1eszslhy&st=d8482nyy&dl=0)
